job-dsls/jobs/reduced_drools_release.groovy

```diff
@@ -0,0 +1,241 @@
+/**
+ * Job that runs the community release pipeline, but only over the reps until & incl. droolsjbpm-integration
+ */
+
+import org.kie.jenkins.jobdsl.Constants
+
+def kieVersion=Constants.KIE_PREFIX
+def baseBranch=Constants.BRANCH
+def releaseBranch="r7.45.0.t20201015"
+def organization=Constants.GITHUB_ORG_UNIT
+def m2Dir = Constants.LOCAL_MVN_REP
+def commitMsg="Upgraded version to "
+
+
+// creation of folder
+folder("reduced-drools-release")
+
+def folderPath="reduced-drools-release"
+
+def redRelease='''
+pipeline {
+    agent {
+        label 'kie-rhel7 && kie-mem24g'
+    }
+    tools {
+        maven 'kie-maven-3.6.3'
+        jdk 'kie-jdk1.8'
+    }
+    stages {
+        stage('CleanWorkspace') {
+            steps {
+                cleanWs()
+            }
+        }
+        stage('Checkout droolsjbpm-build-bootstrap') {
+            steps {
+                checkout([$class: 'GitSCM', branches: [[name: '$baseBranch']], browser: [$class: 'GithubWeb', repoUrl: 'git@github.com:$organization/droolsjbpm-build-bootstrap.git'], doGenerateSubmoduleConfigurations: false, extensions: [[$class: 'RelativeTargetDirectory', relativeTargetDir: 'droolsjbpm-build-bootstrap']], submoduleCfg: [], userRemoteConfigs: [[credentialsId: 'kie-ci-user-key', url: 'git@github.com:$organization/droolsjbpm-build-bootstrap.git']]])
+                dir("${WORKSPACE}" + '/droolsjbpm-build-bootstrap') {
+                    sh  'pwd \\n' +
+                        'git branch \\n' +
+                        'git checkout -b $baseBranch'
+                }
+            }    
+        }
+        stage ('Clone others') {
+            steps {
+                sshagent(['kie-ci-user-key']) {
+                    sh 'sh droolsjbpm-build-bootstrap/script/release/01_cloneBranches.sh $baseBranch'
```

Comment:
so we are cloning all repository list? Where is the preparation of reduced repository list?

Reply:
fixd: look at stage replace repository-list.txt and change branched-7-repository-list.txt